core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -1810,7 +1810,7 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
    * Test the consumer group APIs.
    */
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
-  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17960"))
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
```

Comment:
Since KAFKA-17960 is still open, does this change cause the test to become unstable?

Reply:
The root cause of KAFKA-17960 is already fixed by https://issues.apache.org/jira/browse/KAFKA-18188, so I think it's stable now.

Reply:
Could you please remove `getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17960`  totally?

Reply:
Yes, removed it. Thanks.